web/settings/modulconfigpv.php

```diff
@@ -555,8 +555,8 @@
 										</label>
 									</div>
 									<span class="form-text small">
-										Diese Option aktivieren wenn ein Solaredge SmartMeter verbaut ist welches vorhandene Bestands PV Anlagen erfasst.
-										Das Meter muss an Position 2 konfiguriert sein.
+										Diese Option nur aktivieren, wenn ein weiteres Solaredge SmartMeter verbaut ist, welches z.B. vorhandene Bestands PV Anlagen erfasst.
+										Dieses zusätzliche SmartMeter muss dann als "Zähler 2" / "Position 2" im WE-Konfiguratioonsmenü konfiguriert sein.
```

Comment:
Interessant wäre auch noch die Info, was das eigentlich soll. "Die vom weiteren Zähler gemessene Leistung wird zu der PV-Leistung dieses Wechselrichters hinzuaddiert". Oder sowas in der Art.